components/Navbar.tsx

```diff
@@ -69,14 +92,29 @@ const Navbar = ({ user, open, toggleDrawer }) => {
             Profile
           </Typography>
         </Grid>
-        <Grid xs={2} sx={{ paddingBottom: 1 }}>
+        <Grid
+          xs={2}
+          sx={{ paddingBottom: 1 }}
+          style={{
+            display: 'flex',
+            alignItems: 'center',
+            justifyContent: 'center',
+          }}
```

Comment:
This isn't the recommended approach to handle style, it is inline style, the better way is something like this: https://mui.com/system/styles/basics/#hook-api

and also, I don't think Grid is needed here, this is just simple case for Box/div

Reply:
I think that is legacy?

<img width="290" alt="Screenshot 2023-10-15 at 10 56 13 AM" src="https://github.com/Greenstand/treetracker-web-map-demo/assets/86506971/fe774e58-b4e2-4ec5-949a-72083f4616ab">